osquery/tables/networking/freebsd/process_open_sockets.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+ *  Copyright (c) 2015, Wesley Shields.
```

Comment:
These copyright notices will be overwritten by bots and codemodes to 2014, Facebook as part of the CLA. If we want names/handles to persist past git history we should use a CONTRIBUTORS file.